Fix HTTPS config and redirect #268 (#289)
spiffe/tornjak: examples/tls_mtls/README.md

More details on the configmap can be found [in our config documentation](../../docs/config-tornjak-server.md). 

> One Tornjak server can open two connections simultaneously: HTTP and HTTPS. HTTP is always enabled, and when HTTPS is enabled, the HTTP port reroutes to the HTTPS URL. A configuration that enables TLS is provided in [the configmap in this directory](./tornjak-configmap.yaml). 

Reviewer: ```suggestion
One Tornjak server opens two connections simultaneously: HTTP and HTTPS. HTTP is always enabled, and when HTTPS is enabled, the HTTP port reroutes to the HTTPS port. A configuration that enables TLS is provided in [the configmap](./tornjak-configmap.yaml). 
```

```
server {
  ...
  https {
    port = 10443             # container port for mTLS connection

>     cert = "server/tls.crt"  # mTLS cert

Reviewer: ```suggestion
    cert = "server/tls.crt"  # TLS cert
```

>     key = "server/tls.key"   # mTLS key

Reviewer: ```suggestion
    key = "server/tls.key"   # TLS key
```

>     ca = "users/userCA.crt"  # mTLS CA [Removing this line creates a TLS connection]

Reviewer: ```suggestion
    ca = "users/userCA.crt"  # user CA for mTLS [Removing this line creates a TLS connection]
```

Now that we have opened TLS and mTLS connection, we may make calls. You will need the url to access the endpoints.  If you have deployed locally on Minikube, as in the quickstart, you will need to port-forward the container to localhost. 

### Redirect from HTTP Port

> Notice that with this new configuration, a curl command to the HTTP port gives the URL of the HTTPS port:

Reviewer: ```suggestion
Notice that with this new configuration, a `curl` command to the HTTP port returns a permanent redirect to the URL of the HTTPS port:
```
